static/js/ui_init.js

```diff
@@ -83,7 +105,11 @@ exports.initialize_kitchen_sink_stuff = function () {
         // preventDefault, allowing the modal to scroll normally.
     });
 
-    $(window).resize(_.throttle(resize.handler, 50));
+    $(window).resize(_.throttle(function () {
+        if (should_resize() === true) {
```

Comment:
you don't need to use `=== true` ever; just test for the thing.

Reply:
Oh sorry I messed there, minor mistakes.